scripts/leapfrog/README.md

```diff
@@ -107,6 +107,8 @@ If a leapfrog upgrade is interrupted, it can be resumed. Each major step of the
 leapfrog upgrade process creates a marker file, which will be used to skip
 completed tasks on subsequent runs.
 
+To resume, run scripts/leapfrog/ubuntu14-leapfrog.sh.
```

Comment:
Would be nice to have backticks for the script name, but I'm not holding it back for that. ;)